src/JustSaying/AwsTools/MessageHandling/SnsPolicyDetailsIamExtensions.cs

```diff
@@ -1,12 +1,11 @@
 ﻿using System.Text.Json;
-using System.Text.RegularExpressions;
 using Amazon;
 
 namespace JustSaying.AwsTools.MessageHandling;
 
-internal static class SnsPolicyBuilder
+internal static class SnsPolicyDetailsIamExtensions
```

Comment:
I would be inclined to consolidate the two extension classes into one and then name it something more generic

Reply:
ok